packages/mandelbrot/assets/scss/components/_navigation.scss

```diff
@@ -1,6 +1,40 @@
 .Navigation {
-    display: inline-block;
+    display: flex;
+    position: relative;
     width: 100%;
+    height: 100%;
+    overflow: hidden;
+}
+
+.Navigation-panel {
+    @include border-inline(end, 1px solid $color-frame-border);
+
+    width: 240px;
+    height: 100%;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
```

Comment:
Since Safari/iOS 13 this is not required anymore. I guess we can remove it considering the [adoption rate](https://appleinsider.com/articles/20/12/16/ios-14-is-running-on-72-of-all-devices-surpassing-ios-13-adoption-rates) and the fact it used to be buggy on elements where scroll would be dynamically added/removed based on the inner content.

Reply:
Yeah makes sense, especially since our browserslist config also should only support last 2 versions. I removed it from everywhere we had used it before.